umbrella/README.md

```diff
@@ -0,0 +1,95 @@
+# `polkadot-sdk`
+
+`polkadot-sdk` is an umbrella crate re-exporting all other published crates, except for the example and fuzzing, plus some of the runtime crates like Rococo. Consider it the entry to the whole Polkadot and Substrate ecosystem.
+
+`polkadot-sdk` aims to make the SDK more approachable - the entire development environment made available with one dependency. More importantly, it guarantees the right combination of crate versions, thus third-party tools for selecting compatible crate versions are no longer needed.
+
+## Usage
+
+The re-exported crates are grouped under the following feature sets.
+
+- `std`: for enabling `std` support of the enabled features
+- `experimental`: for experimental features
+- `node`: for node implementation
+- `runtime`: for runtime implementation
+- `runtime-benchmarks`: for benchmarking runtimes
+- `runtime-full`: for more comprehensive runtime implementation?
+- `serde`: for `serde` en/decoding support
+- `tuples-96`: required by runtimes with more than 64 pallets
+- `try-runtime`: for cli support?
+- `with-tracing`: for tracing support
+
+Choose which ones to enable based on the needs. For example, when building a runtime with more than 64 pallets, benchmarking needs and optional tracing, the `Cargo.toml` may contain
+
+```toml
+[dependencies]
+polkadot-sdk = { version = "0.12.0", features = ["runtime", "tuples-96"] }
+
+[features]
+runtime-benchmarks = ["polkadot-sdk/runtime-benchmarks"]
+with-tracing = ["polkadot-sdk/with-tracing"]
+```
+
+```shell
+# build enabling runtime-benchmarks and with-tracing
+$ cargo build --features "runtime-benchmarks,with-tracing"
+```
+
+## Pro Tips
+
+* fine-grained control over dependencies
+* any suggestions?
+
+In addition to the features above, each re-exported crate is feature-gated individually so that one can control exactly which crates to include. In the contrived example below, only the `frame-support` and `frame-system` crates are needed.
```

Comment:
Some context:

This is done to reduce build time in cases where only a small subset of dependencies is needed. The less dependencies you enable - the faster it builds.